docs/guide/writing-tasks.md

```diff
@@ -250,10 +250,10 @@ the same organization cannot re-use them. `qwerty239abc` from the example above
 variable, it's just an internal ID. No one can brute force your secrets from such ID. In addition, Cirrus CI doesn't know
 a relation between an encrypted variable and a repository for which the encrypted variable was created.
 
-## Matrix Modification
+## Matrix Modifier
```

Comment:
But the state is modified because of modifications made.
IMO, this makes more sense to revert the title.  
```suggestion
## Matrix Modification
```

Reply:
Actually, making it plural might be a good compromise...